<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Travis Cobbs</author>
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LPub Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Default author name:</string>
         </property>
         <property name="buddy">
          <cstring>authorName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="authorName"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="ldrawBox">
         <property name="toolTip">
          <string>This directory contains the LDraw parts directory</string>
         </property>
         <property name="title">
          <string>LDraw Root Directory</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ldrawPath">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseLDraw">
            <property name="toolTip">
             <string>Use directory dialog to find LDraw directory</string>
            </property>
            <property name="whatsThis">
             <string>This lets you find the LDraw directory</string>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="pliBox">
         <property name="toolTip">
          <string>This file controls Part List parts orientation and size</string>
         </property>
         <property name="title">
          <string>Use PLI orientation/size control file</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pliName">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browsePli">
            <property name="toolTip">
             <string>Use directory dialog to find the PLI file</string>
            </property>
            <property name="whatsThis">
             <string>Use directory dialog to find the PLI file</string>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="fadeStepBox">
         <property name="title">
          <string>Enable Fade Steps</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>29</x>
            <y>16</y>
            <width>111</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>Default fade color</string>
          </property>
         </widget>
         <widget class="QComboBox" name="fadeStepColorsCombo">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>44</y>
            <width>281</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="fadeStepColorLabel">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>45</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Default Units</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QRadioButton" name="Centimeters">
            <property name="text">
             <string>Centimeters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Inches">
            <property name="text">
             <string>Inches</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="projectsFolderBox">
         <property name="toolTip">
          <string>This directory contains the LDraw parts directory</string>
         </property>
         <property name="title">
          <string>Default Projects Folder</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="_2">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="projectsFolder">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseProjectsFolder">
            <property name="toolTip">
             <string>Use directory dialog to find LDraw directory</string>
            </property>
            <property name="whatsThis">
             <string>This lets you find the LDraw directory</string>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRendering">
      <attribute name="title">
       <string>Rendering</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>116</width>
         <height>22</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Preferred renderer:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="preferredRenderer">
       <property name="geometry">
        <rect>
         <x>134</x>
         <y>20</y>
         <width>321</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>L3P</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LDGLite</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LDView</string>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="l3pBox">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>260</y>
         <width>441</width>
         <height>178</height>
        </rect>
       </property>
       <property name="title">
        <string>L3P is installed</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>POV-RAY</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="povrayPath"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="browseL3P">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="browsePOVRAY">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="l3pPath"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>L3P</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QGroupBox" name="lgeoBox">
          <property name="toolTip">
           <string>This library provides higher quality rendering models for POV RAY</string>
          </property>
          <property name="title">
           <string>LGEO is available</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lgeoPath">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="browseLGEO">
             <property name="toolTip">
              <string>Use directory dialog to find LGEO directory</string>
             </property>
             <property name="whatsThis">
              <string>This lets you find the LGEO directory</string>
             </property>
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="ldgliteBox">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>65</y>
         <width>441</width>
         <height>78</height>
        </rect>
       </property>
       <property name="title">
        <string>LDGLite is installed</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ldglitePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseLDGLite">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="ldviewBox">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>162</y>
         <width>441</width>
         <height>77</height>
        </rect>
       </property>
       <property name="title">
        <string>LDView is installed</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ldviewPath"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseLDView">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabPublishing">
      <attribute name="title">
       <string>Publishing</string>
      </attribute>
      <widget class="QTextEdit" name="publishDescriptionEdit">
       <property name="geometry">
        <rect>
         <x>28</x>
         <y>241</y>
         <width>421</width>
         <height>191</height>
        </rect>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="publishLogoBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>125</y>
         <width>451</width>
         <height>68</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>This directory contains the LDraw parts directory</string>
       </property>
       <property name="title">
        <string>Title Page Logo</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="_5">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="publishLogoPath">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browsePublishLogo">
          <property name="toolTip">
           <string>Use directory dialog to find LDraw directory</string>
          </property>
          <property name="whatsThis">
           <string>This lets you find the LDraw directory</string>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="publishDescriptionBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>451</width>
         <height>241</height>
        </rect>
       </property>
       <property name="title">
        <string>Publish Description</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="publishTOC">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>17</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate PDF Table of Contents</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>55</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Default URL:</string>
       </property>
       <property name="buddy">
        <cstring>authorName</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="publishURL">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>55</y>
         <width>361</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Default Email:</string>
       </property>
       <property name="buddy">
        <cstring>authorName</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="publishURL_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>361</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="publishCopyright">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>19</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Print Copyright statement</string>
       </property>
      </widget>
      <zorder>publishDescriptionBox</zorder>
      <zorder>publishDescriptionEdit</zorder>
      <zorder>publishLogoBox_3</zorder>
      <zorder>publishTOC</zorder>
      <zorder>label_15</zorder>
      <zorder>publishURL</zorder>
      <zorder>label_17</zorder>
      <zorder>publishURL_2</zorder>
      <zorder>publishCopyright</zorder>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
